TRB_label_assist/models.py

```diff
@@ -351,6 +351,7 @@ def predict(self, test_df):
         elif self.loc_type == 'end':
             bins = self.sim_model.bins
 
+        bins = {int(key):value for key,value in bins.items()}
```

Comment:
```suggestion
        # This looks weird but works
        # >>> x = [(1, 'a'), (2, 'b'), (3, 'c')]
        # >>> {int(key):value for key,value in x}
        # {1: 'a', 2: 'b', 3: 'c'}
        bins = {int(key):value for key,value in bins.items()}
```

Reply:
```
        # bins = { '1': [ .. ....], 
        #          '2': [......], 
        #          '3': [.......] ....}
        # the  code converts this to 
        # bins = { 1: [ .. ....], 
        #          2: [......], 
        #          3: [.......] ....}
        ```

Reply:
right. I'm just suggesting that you add a comment there indicating that it works since it looks weird to create a dict without a list (I understand that it works because of the iterable, but let's make life easier for others in the future).

Reply:
I am unable to get what's expected here. I wrote an example there explaining what it does. I am now adding why it works there. Let me know if I missed something.

Reply:
@humbleOldSage I have a suggested change right here.
https://github.com/e-mission/e-mission-eval-private-data/pull/37#discussion_r1388696448
suggested 2 weeks ago

All you have to do is to incorporate the suggested change. I am not sure what is not clear here.